### DISCUSSION TITLE
Attestation-Based Data Duplicator

### DISCUSSION THREAD

**Discussion by aagbotemi** [ethereum-optimism/ecosystem-contributions#175] [Builds]
Hey from the Duplikaat Team,

we are working at the solution for the Attestation-Based Data Duplicator idea as described at https://contribute.optimism.io/issue/attestation-data-duplicator

We already implemented some projects on Ethereum, Starknet and Polygon and now we want to create a developer tool on Optimism.

We want to implement these points:

- Addresses that were part of the initial ETH genesis allocation
- Addresses that have held some NFT for more than X amount of time
- Addresses that have donated to GitCoin rounds
- Addresses that have received POAPs from important events

Everyone is welcome to help to discuss this new idea with us!

**Comment by must479** [comment]
Addresses that have received POAPs from important events
https://collectors.poap.xyz/token/5798768

My address sangbejo19.eth

**Comment by must479** [comment]
Yes

Pada Sen, 22 Jul 2024, 2.59 PM, Must479 ***@***.***> menulis:
